src/Microsoft.Azure.WebJobs.Host/Bindings/ConverterManager.cs

```diff
@@ -32,15 +94,31 @@ private static string DefaultByteArray2String(byte[] bytes)
             return typeof(TSrc).FullName + "|" + typeof(TDest).FullName + "|" + typeof(TAttribute).FullName;
         }
 
+        public void AddConverterBuilder<TSrc, TDest, TAttribute>(
+            Func<Type, Type, Func<object, object>> converterBuilder)
+            where TAttribute : Attribute
+        {
+            if (typeof(OpenType).IsAssignableFrom(typeof(TSrc)) ||
+                typeof(OpenType).IsAssignableFrom(typeof(TDest)))
+            {
+                AddOpenConverter<TSrc, TDest, TAttribute>(converterBuilder);
+                return;
+            }
+
+            string key = GetKey<TSrc, TDest, TAttribute>();
+            _funcsWithAttr[key] = converterBuilder;
+        }
+
         public void AddConverter<TSrc, TDest, TAttribute>(FuncConverter<TSrc, TAttribute, TDest> converter)
             where TAttribute : Attribute
-        {           
+        {
             string key = GetKey<TSrc, TDest, TAttribute>();
-            _funcsWithAttr[key] = converter;            
+            _funcsWithAttr[key] = converter;                        
         }
 
         // Return null if not found. 
-        private FuncConverter<TSrc, TAttribute, TDest> TryGetConverter<TSrc, TAttribute, TDest>()
+        private FuncConverter<TSrc, TAttribute, TDest> TryGetConverter<TSrc, TAttribute, TDest>(
```

Comment:
Generally non async TryGet methods return a bool and return their value as an output parameter.